src/server/services/model.service.ts

```diff
@@ -523,13 +538,21 @@ export const getModelsRaw = async ({
     modelVersionWhere.push(Prisma.sql`cm."modelVersionId" = mv."id"`);
   }
 
+  const includeDetails = !!include?.includes('details');
+
   const modelQuery = Prisma.sql`
     ${queryWith}
     SELECT
       m."id",
       m."name",
+      ${includeDetails ? Prisma.raw('m."description",') : Prisma.empty}
```

Comment:
Is there any reason why we don't group this sub-query as a single thing? 
```
includeDetails ? Prisma.raw(`m.desc, m.poi, etc, etc`) : Prisma.empty
```

Reply:
Agreed.